src/rospkg/environment.py

```diff
@@ -39,9 +39,10 @@
 # Enviroment Variables
 
 # Global, usually set in setup
-ROS_ROOT         = "ROS_ROOT"
-ROS_PACKAGE_PATH = "ROS_PACKAGE_PATH"
-ROS_HOME         = "ROS_HOME"
+ROS_ROOT            = "ROS_ROOT"
+ROS_PACKAGE_PATH    = "ROS_PACKAGE_PATH"
+ROS_HOME            = "ROS_HOME"
+ROS_RPP_NO_DEFAULTS = "ROS_RPP_NO_DEFAULTS"
```

Comment:
The abbreviation `RPP` is not used anywhere. Therefore I would suggest sticking to a more verbose name: `ROS_PACKAGE_PATH_NO_DEFAULTS`.